### DISCUSSION TITLE
Toggle Dark/Light mode on www.pymc.io

### DISCUSSION THREAD

**Discussion by hectormz** [pymc-devs/pymc3#5899] [Q&A]
I have viewed the new version of the website in both dark and light mode, but can't seem to toggle between the two. I'm assuming that a preference for dark mode is being read from my browser/computer. But I find the website much harder to read in dark mode. 

Is there a setting I'm not seeing, or is it a setting that can be added? 

@OriolAbril I think you're heavily involved with the website and might have an idea.

**Comment by OriolAbril** [comment]
We should disable the dark theme as none of our content is ready for it. We need to modify:

* https://github.com/pymc-devs/pymc/blob/main/docs/source/conf.py
* https://github.com/pymc-devs/pymc.io/blob/main/conf.py
* https://github.com/pymc-devs/pymc-examples/blob/main/examples/conf.py

and add `"default_mode": "light"` to the `html_context` dict as indicated in https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/configuring.html#configure-default-mode. We are already using custom elements for navbar end which is why the mode switcher doesn't appear and why you can't change from dark to light, so we don't need to do the suggestion in the tip box to force light theme to always be used.
